serverless.js

```diff
@@ -1,7 +1,21 @@
+const assert = require('assert');
 const { paramCase } = require('param-case');
+
 const pkg = require('./package.json');
 
-const { NODE_ENV = 'development' } = process.env;
+const {
+  AWS_REGION = 'us-east-1',
+  SLS_STAGE = 'development',
+  BASE_URL,
+  AWS_ACM_CERTIFICATE_ARN,
+  NODE_ENV = 'development',
+} = process.env;
+
+assert.ok(BASE_URL, 'process.env.BASE_URL not defined');
```

Comment:
I think these checks should be in a conditional `NODE_ENV==='production'`? Otherwise `sls package` etc locally will fail

Reply:
We have two options
* Validate when production
* Create the custom code only when the variables are present

Reply:
The first is much easier and good enough I guess